lib/pages/chat/events/message.dart

```diff
@@ -86,7 +86,7 @@ class Message extends StatelessWidget {
             ownMessage ? MainAxisAlignment.end : MainAxisAlignment.start;
 
         final displayEvent = event.getDisplayEvent(timeline);
-        final noBubble = { MessageTypes.Video, MessageTypes.Sticker}
+        final noBubble = { MessageTypes.Sticker}
```

Comment:
why not keep no bubble for it?

Reply:
The condition is used for checking padding, so I make the video bubble the same as the image bubble.
<img width="430" alt="Screenshot 2023-08-11 at 10 39 00" src="https://github.com/linagora/twake-on-matrix/assets/43041967/b182be53-f256-4f8b-984c-ae43481bb5a2">